Examples/Python/tests/helpers/__init__.py

```diff
@@ -66,6 +66,13 @@
 except ImportError:
     pythia8Enabled = False
 
+try:
+    import acts.examples
+
+    hashingSeedingEnabled = hasattr(acts.examples, "SeedingAlgorithmHashing")
```

Comment:
That way you could test for the existence of the module here.

Reply:
I added a `examples/hashing.py` file, allowing to test for the existence of the module.